guard/src/rules/path_value.rs

```diff
@@ -122,41 +124,22 @@ impl Path {
         copy.push_str(part);
         Path(copy, self.1.clone())
     }
-
-    pub(crate) fn extend_str_with_location(&self, part: &str, loc: Location) -> Path {
-        let mut copy = self.0.clone();
-        copy.push('/');
-        copy.push_str(part);
-        Path(copy, loc)
-    }
-
-    pub(crate) fn extend_string(&self, part: &String) -> Path {
-        self.extend_str(part.as_str())
+    //
+    // pub(crate) fn extend_str_with_location(&self, part: &str, loc: Location) -> Path {
+    //     let mut copy = self.0.clone();
+    //     copy.push('/');
+    //     copy.push_str(part);
+    //     Path(copy, loc)
+    // }
```

Comment:
Why is this chunk commented out?

Reply:
Compiler was complaining about unused code, and i commented it out and ran the test suite + did manual testing to make sure it wasnt actually being used...Forgot to delete it. Good catch